packages/core/src/external/mpi/patient-blocker.ts

```diff
@@ -0,0 +1,20 @@
+import { GenderAtBirth, PatientDataMPI } from "./patient";
+
+export type PatientBlock = {
+  cxId?: string;
+  facilityIds?: string[];
+  data?: {
+    dob?: string;
+    genderAtBirth?: GenderAtBirth;
+    firstNameInitial?: string;
+    lastNameInitial?: string;
+  };
+};
```

Comment:
This looks very much like API's `Patient` 😄 

I think the convo on Slack will impact this, but it would be good to avoid yet another Patient type.

Reply:
good point. I didnt even notice that. Maybe there is a way around this issue.